docker-bake.hcl

```diff
@@ -15,8 +15,12 @@ variable "SEMVER" {
   default = "v0.0.0"
 }
 
-variable "GITCOMMIT" {
-  default = "dev"
+variable "GIT_SHA" {
+  default = "$GIT_SHA"
+}
+
+variable "GIT_SHORT_SHA" {
+  default = "$GIT_SHORT_SHA"
```

Comment:
These defaults are designed to fail (via invalid tag format) if someone tries to bake a release target without exporting `GIT_SHA` or `GIT_SHORT_SHA` env variables
```
docker buildx bake internal-release
[+] Building 0.1s (1/1) FINISHED                                                                                                                                                                                docker:desktop-linux
 => [internal] load local bake definitions                                                                                                                                                                                      0.0s
 => => reading docker-bake.hcl 5.14kB / 5.14kB                                                                                                                                                                                  0.0s
ERROR: invalid tag "ghcr.io/eigenda-traffic-generator:$GIT_SHA": invalid reference format
```
without a default, the build will still fail, but error message is harder to grok what the issue is.
```
docker buildx bake internal-release
[+] Building 0.1s (1/1) FINISHED                                                                                                                                                                                docker:desktop-linux
 => [internal] load local bake definitions                                                                                                                                                                                      0.0s
 => => reading docker-bake.hcl 5.14kB / 5.14kB                                                                                                                                                                                  0.0s
ERROR: invalid tag "ghcr.io/eigenda-traffic-generator:": invalid reference format
```

Reply:
I'd prob add this as a comment above the variables themselves for documentation. Otherwise I can see myself reading this and being super confused wondering where/how this variable is defined and used.